simpleflow/swf/executor.py

```diff
@@ -18,10 +18,15 @@
     task,
 )
 from simpleflow.activity import Activity
+from simpleflow.base import Submittable
 from simpleflow.history import History
 from simpleflow.swf import constants
 from simpleflow.swf.helpers import swf_identity
 from simpleflow.swf.task import ActivityTask, WorkflowTask
+from simpleflow.task import (
```

Comment:
I think it's used publicly so not a good idea, we will have to keep some references directly there (in botify private code we directly use that for instance)